aws_lambda_powertools/utilities/parser/models/kafka.py

```diff
@@ -0,0 +1,84 @@
+import base64
+import logging
+from binascii import Error as BinAsciiError
+from datetime import datetime
+from typing import Dict, List, Optional, Type, Union
+
+from pydantic import BaseModel, validator
+
+from aws_lambda_powertools.utilities.parser.types import Literal
+
+SERVERS_DELIMITER = ","
+
+logger = logging.getLogger(__name__)
+
+
+def _base64_decode(value: str) -> bytes:
+    try:
+        logger.debug("Decoding base64 Kafka record item before parsing")
+        return base64.b64decode(value)
+    except (BinAsciiError, TypeError):
+        raise ValueError("base64 decode failed")
+
+
+def _bytes_to_string(value: bytes) -> str:
+    try:
+        return value.decode("utf-8")
+    except (BinAsciiError, TypeError):
+        raise ValueError("base64 UTF-8 decode failed")
```

Comment:
As we are adding new features and always improving our code, what do you think about moving these 2 functions to the shared functions [file](https://github.com/awslabs/aws-lambda-powertools-python/blob/develop/aws_lambda_powertools/shared/functions.py)?

Reply:
sure, np